test/L1Escrow.t.sol

```diff
@@ -154,7 +154,92 @@ contract EscrowTest is Setup {
         assertEq(vault.balanceOf(address(mockEscrow)), 0);
     }
 
-    function test_bridgeAsset_maxDepositLimit(uint256 _amount) public {
+    function test_bridgeAsset_escrowDepositLimit(uint256 _amount) public {
+        _amount = bound(_amount, minFuzzAmount, maxFuzzAmount);
+        address counterPart = l1Deployer.getL2EscrowAddress(
+            l2RollupID,
+            address(asset)
+        );
+        mockEscrow = deployMockL1Escrow();
+
+        vm.expectRevert();
```

Comment:
some comments would be helpful :)

// reverts because caller does is not allowed to change the deposit limit

Reply:
https://github.com/yearn/yearn-stb/pull/17/commits/db539770778790eb87070299e02103f0c3c9af82